src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/controller/CoordinatorController.kt

```diff
@@ -248,10 +248,8 @@ class CoordinatorController(
      *  - Save the changes and return the the plan UUID and version number
      *  - Handle any exceptions or edge cases (i.e plan or version not found, invalid sign type, countersigning failures))
      */
-    return PlanVersionResponse(
-      planId = planUuid,
-      planVersion = body.sentencePlanVersion,
-    )
+    return planService.countersignPlan(planUuid, countersignPlanRequest)
```

Comment:
should the TODO above this be removed?

Reply:
yes. hopefully you'll agree that I've to-done it.

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/113/commits/dfe7e35312c63d90e3a0e50872423612026d0fcd